sympy/printing/codeprinter.py

```diff
@@ -69,6 +69,18 @@ class CodePrinter(StrPrinter):
     # may be supported
     # function_to_rewrite : (function_to_rewrite_to, iterable_with_other_functions_required)
     _rewriteable_functions = {
+            'cot': ('tan', []),
+            'csc': ('sin', []),
+            'sec': ('cos', []),
+            'acot': ('atan', []),
+            'acsc': ('asin', []),
+            'asec': ('acos', []),
+            'coth': ('tanh', []),
+            'csch': ('sinh', []),
+            'sech': ('cosh', []),
```

Comment:
This one doesn't work
```
In [1]: lambdify(x, sech(x))(1)
---------------------------------------------------------------------------
RecursionError
```
It would work if changed to `sinh` but actually I think the proper rewrite rule should be added to sech instead.
```
In [2]: sech(x).rewrite(cosh)
Out[2]: sech(x)

In [3]: sech(x).rewrite(sinh)
Out[3]: 
   1   
───────
cosh(x)
```

Reply:
Done!